app/src/androidTest/java/com/github/sdp_begreen/begreen/fragments/MapFragmentTest.kt

```diff
@@ -35,96 +39,83 @@ class MapFragmentTest {
      */
     companion object {
 
-        private val db: DB = Mockito.mock(DB::class.java)
+        private val db: DB = mock(DB::class.java)
```

Comment:
```suggestion
        private val db: DB by lazy { mock() }
```
I just figured out that we can use lazy for this. For the tests it  seems pretty useless but for the main code we should think about it. Also kotlin can infer the type, so we can just write movk() =)

Reply:
Well found, I'll try to use that from now on!